subprojects/gradle/instrumentation-tests-test-fixtures/src/main/kotlin/com/avito/instrumentation/report/listener/FakeTestReporter.kt

```diff
@@ -0,0 +1,11 @@
+package com.avito.instrumentation.report.listener
+
+import com.avito.runner.service.model.TestCase
+import org.funktionale.tries.Try
+import java.io.File
+
+class FakeTestReporter : TestReporter() {
```

Comment:
We use two naming patterns - fake and stub. Is there any difference?

Reply:
In that case, it is an EmptyTestReporter

Reply:
Empty and Stub are the same thing )

Reply:
https://stackoverflow.com/a/346440/981330

Reply:
https://github.com/avito-tech/avito-android/discussions/679
Created discussion